config/locales/en.yml

```diff
@@ -377,6 +377,7 @@ en:
       authenticating_as: Authenticating as
       authenticate: Authenticate
       no_webauthn_devices: You don't have any security devices enabled
+      safari_note: It looks like you are using Safari. Due to limitations within Safari, you will be unable to authenticate using this browser. Please use a different browser.
```

Comment:
I was uncertain what to link to. Do you think it would make sense to link to this: https://bugs.webkit.org/show_bug.cgi?id=171934?

Reply:
Yep, that's the one I was thinking of (it would've been nicer of me to link it, but thus is hurrying).

Reply:
@jchestershopify are you intending to link https://bugs.webkit.org/show_bug.cgi?id=171934 in the docs? I'd prefer to not directly link to the bug report but our docs

Reply:
@jenshenny In that case do you think it would be better to add something like "Refer to the RubyGems docs to find out more about this." instead of linking directly?

Reply:
Yeah something like refer to our docs + I would link the page to would give the info. IIRC it would be `https://guides.rubygems.org/using-webauthn-mfa-in-command-line` from the docs PR.

Reply:
So long as we make it clear that the underlying cause is not due to RubyGems.org, I'm not hugely fussed about whether it's here or by reference to the docs ([in the latest commit I mention in both the setup and CLI pages](https://github.com/rubygems/guides/pull/332/commits/9537ce77e01399e707bf3ab086ebd3a4b618ba5a)).